### PR TITLE
Install VTK and ITK to reduce ResourceCompile length

### PR DESCRIPTION
**Pull request by Sunderlandkyl**
- Avoids errors such as: "The specified task executable "rc.exe" could not be run. The filename or extension is too long"

### CODE REVIEW

File: SuperBuild/External_ITK.cmake [hunk 1]
```diff
@@ -3,17 +3,20 @@ IF(ITK_DIR)
...
 ELSE()
+
+  SET (PLUS_ITK_VERSION_MAJOR 5)
+  SET (PLUS_ITK_VERSION_MINOR 0)
   # ITK has not been built yet, so download and build it as an external project
   SetGitRepositoryTag(
     itk
     "${GIT_PROTOCOL}://itk.org/ITK.git"
-    "v5.0rc02"
+    "v${PLUS_ITK_VERSION_MAJOR}.${PLUS_ITK_VERSION_MINOR}rc02"
```
Comment (adamrankin) [inline SuperBuild/External_ITK.cmake]: I wouldn't do it quite like this. For example, there is unlikely to be 5.1rc02 etc...
You could change it to another variable like PLUS_ITK_VERSION_PATCH

Comment (Sunderlandkyl) [reply SuperBuild/External_ITK.cmake]: Ok, changed.

**Comment by adamrankin** [comment]
Lgtm

**Comment by lassoan** [comment]
Thank you, this is awesome. It will make our life much easier.

**Pull request merged**